app/filter.py

```diff
@@ -277,6 +288,28 @@ def update_styling(self, soup) -> None:
         except AttributeError:
             pass
 
+        # Fix body max with on images tab
+        style = soup.find('style')
+        div = soup.find('div', attrs={'class': "n692Zd"})
+        if style and div and not self.mobile:
+            css = style.string
+            css_html_tag = (
+                'html{'
+                'font-family: Roboto, Helvetica Neue, Arial, sans-serif;'
+                'font-size: 14px;'
+                'line-height: 20px;'
+                'text-size-adjust: 100%;'
+                'color: #3c4043;'
+                'word-wrap: break-word;'
+                'background-color: #fff;'
+                '}'
```

Comment:
Why is this necessary? It doesn't seem to affect the page from what I can tell, and things like `background-color` and `color` are being ignored altogether. I tried removing it and couldn't tell any difference.

Reply:
The colors and background colors can be removed, the others are needed to have a uniform styling across the tabs (It is mainly needed to fix the text formatting). It is only used in the images tab when the view image option is disabled!